src/board.spec.js

```diff
@@ -0,0 +1,14 @@
+import { expect } from 'chai';
+import Board from './board'
+
+const matrix = [[true, true , true], [true, true, true], [true, true, true]]
+
+describe('Board', () => {
+  it('should be created', () => {
+    var board = new Board(matrix)
+    board.initialize()
+    board.print()
+    expect(board).to.be.a('object')
+  })
+
```

Comment:
Add more test cases, including print. 

You can actually pass to the board.print the method that outputs strings to the console if you want to mock it (you have to mock it) :+1: